lib/hyrax/acl/custom_queries/find_access_control.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Hyrax
+  module Acl
+    module CustomQueries
+      # @example
+      #   Hyrax.custom_queries.find_access_control_for(resource: resource)
+      class FindAccessControl
+        def self.queries
+          [:find_access_control_for]
+        end
+
+        def initialize(query_service:)
+          @query_service = query_service
+        end
+
+        attr_reader :query_service
+        delegate :resource_factory, to: :query_service
+
+        def find_access_control_for(resource)
```

Comment:
what was the problem with the named param?

it's not tremendously important, but `query_name(resource: resource)` is the patter elsewhere in Valkyrie, so it's slightly preferable to follow that pattern.

Reply:
When I tried running tests it seemed to be passing a hash as a first argument instead of passing the named param properly, giving an error like

```
ArgumentError: wrong number of arguments (given 1, expected 0; required keyword: resource)
```

(where the given argument is `{resource: blah}`)

I tried looking into the valkyrie adapter to see if I could figure out why this was happening but couldn’t track it down.